cmd/controller-manager/app/controller_manager.go

```diff
@@ -247,14 +248,20 @@ func Run(controllerManagerOptions *options.ControllerManagerServer) error {
 		return err
 	}
 
-	klog.V(5).Infof("Using namespace %v for leader election lock", controllerManagerOptions.LeaderElectionNamespace)
+	// create config for interacting with coordination.k8s.io group
+	coordinationClient, err := v1coordination.NewForConfig(k8sKubeconfig)
```

Comment:
not really sure what this api group is for, it might just be they're trying to move things out of the default group. They had the initializer for it so I'm not too worred.

Reply:
looks like leader election leases.

I don't think we have a test for that.